### DISCUSSION TITLE
tabcontrol do not show drop down arrow

### DISCUSSION THREAD

**Discussion by yanglei629** [HandyOrg/HandyControl#1193] [Q&A] [answered]
Hi,i do not want to show the drop down arrow in tabcontrol,but i do not find the property to set it.

##snapshot

![image](https://user-images.githubusercontent.com/81857996/170513342-1afc01f6-becc-42f7-b301-58b4191d4624.png)

**Comment by ghost1372** [comment, marked as answer]
**ShowOverflowButton**
https://ghost1372.github.io/handycontrol/extend_controls/tabControl/

**Comment by yanglei629** [comment]
thanks,it is work.
